imgur-takeover.js

```diff
@@ -0,0 +1,44 @@
+// The Tasks
+//
+// Create a function that changes the navbar to a different color.
```

Comment:
Make sure you're reading directions -- create a function for each of these tasks ;)